invenio_rdm_records/services/config.py

```diff
@@ -71,6 +74,12 @@ def has_doi(record, ctx):
     return "doi" in pids
 
 
+def is_iiif_conpatible(file_, ctx):
+    """Determine if a file is IIIF compatible."""
+    file_ext = splitext(file_.key)[1].replace(".", "").lower()
```

Comment:
Will the filename always have the extension? Basically, does the `[1]` always work?

Reply:
I think it should be alright https://docs.python.org/3/library/os.path.html#os.path.splitext